PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs

```diff
@@ -6,6 +6,51 @@ namespace PrincessBrideTrivia.Tests;
 public class ProgramTests
 {
     [TestMethod]
+public void IncorrectQuestions_AreRecordedWhenAnsweredWrong()
+{
+    // Arrange
+    Question question1 = new()
+    {
+        Text = "What is my name?",
+        Answers = new string[] { "Chris", "Bill", "Frank" },
+        CorrectAnswerIndex = "1"
+    };
+
+    List<Question> incorrectQuestions = new List<Question>();
+
+    // Act
+    bool result = Program.AskQuestion(question1);
+    if (!result)
+    {
+        incorrectQuestions.Add(question1);
+    }
+
+    // Assert
+    Assert.AreEqual(1, incorrectQuestions.Count); 
+}
+
+[TestMethod]
+public void IncorrectQuestion_IsReAsked()
+{
+    // Arrange
+    Question question1 = new()
```

Comment:
```suggestion
    Question question = new()
```
nit: no need to add a 1 if there is only one instance of question in your scope